test/e2e/tests/multiple-transactions.spec.js

```diff
@@ -71,27 +95,50 @@ describe('Confirm transactions', function () {
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
 
-        // confirm multiple transactions
-        await driver.waitForSelector({
-          text: 'Reject 4 transactions',
-          tag: 'a',
+        // initiates a transaction from the dapp
+        await openDapp(driver);
+        // creates first transaction
+        await driver.clickElement({
+          text: 'Send EIP 1559 Transaction',
+          tag: 'button',
         });
-        await driver.clickElement({ text: 'Reject', tag: 'button' });
-        await driver.waitForSelector({
-          text: 'Reject 3 transactions',
-          tag: 'a',
+        await driver.waitUntilXWindowHandles(3);
+        const windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
+        const confirmation = windowHandles[2];
+        await driver.switchToWindowWithTitle('E2E Test Dapp', windowHandles);
```

Comment:
Very minor, but another switch here but I assume we already have the handle in `windowHandles[1]`?